common/functions/src/scalars/dates/number_function.rs

```diff
@@ -318,7 +318,7 @@ impl NumberResultFunction<u16> for ToMonday {
     const IS_DETERMINISTIC: bool = true;
 
     fn return_type() -> Result<DataType> {
-        Ok(DataType::Date16)
+        Ok(DataType::UInt16)
```

Comment:
This is the logical data_type, must be Date16 for toMonday.

Reply:
Ok i should go back